### PR TITLE
add getDijit promise helper, and test case

### PR DESCRIPTION
**Pull request by dylans**
This adds a promise helper to simplify the boilerplate for writing Dijit functional tests.

I wasn't sure how to properly write the test for this without pulling in Dijit, so I wrote the initial test against the CDN which is probably wrong, so I could use guidance on that. In that context, I also wasn't sure what the appropriate values would be for setting timeouts, so I could use guidance there as well.

### CODE REVIEW

**Comment by jason0x43** [comment]
This would be even more useful if it provided some high-level helpers for interacting with Dijits, like for selecting entries in a dropdown or picking a date on a calendar.

This module might also fit better in Dijit itself rather than Leadfoot. That would ensure it stayed in sync with Dijit development, which it will likely need to track more closely than Leadfoot developemnt, particularly if more complex helpers are included.

File: helpers/dijit.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	 *
+	 * @memberOf module:leadfoot/helpers/dijit#
+	 *
+	 * @param {string} widgetId
+	 * An id for a Dijit reference.
+	 *
+	 * @param {string} attachPoint
+	 * A string representing a Dijit childNode.
+	 */
+	getNode: function (widgetId, attachPoint) {
```
Comment (dylans) [inline helpers/dijit.js]: Should I call this getNodeById? Should there be other mechanisms to get a node reference, e.g. by xpath or css selector?

File: helpers/dijit.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		return function (ignoredValue, setContext) {
+			// TODO: for the next line, it is better to say this.parent or this.session ?
```
Comment (dylans) [inline helpers/dijit.js]: See my question here... both work, but is either more correct?

**Comment by dylans** [comment]
I've finally made some progress on simplifying this helper to the point where the API I had now matches my original goals for making it simple to use. Up next, I plan to address @neonstalwart's API suggestions for matching the registry API, and then perhaps @jason0x43's idea of helpers that are more specific by widget type, though I would probably want to land this somewhere (either in Leadfoot or in a separate package, before tackling the latter).

**Comment by dylans** [comment]
I've moved this to https://github.com/dylans/dijit-intern-helper , though it may move under sitepen or it could move under leadfoot or as a separate intern package if that is preferred. I'm fine with whatever, but the usefulness of this for Dijit users should be made available, rather than buried in a PR, hence my moving it to a separate package for now.

**Pull request closed without merging**